src/app/zone-ingresses/views/ZoneIngressListView.vue

```diff
@@ -42,29 +42,29 @@
               data-testid="zone-ingress-collection"
               :headers="[
                 { label: 'Name', key: 'name' },
-                { label: 'Address', key: 'addressPort' },
-                { label: 'Advertised address', key: 'advertisedAddressPort' },
+                { label: 'Address', key: 'socketAddress' },
+                { label: 'Advertised address', key: 'advertisedSocketAddress' },
                 { label: 'Status', key: 'status' },
                 { label: 'Details', key: 'details', hideLabel: true },
               ]"
               :page-number="1"
               :page-size="100"
               :total="data?.total"
-              :items="data ? transformToTableData(data.items) : undefined"
+              :items="data?.items"
               :error="error"
               :empty-state-message="t('common.emptyState.message', { type: 'Zone Ingresses' })"
               :empty-state-cta-to="t('zone-ingresses.href.docs')"
               :empty-state-cta-text="t('common.documentation')"
               :is-selected-row="(row) => row.name === route.params.zoneIngress"
               @change="route.update"
             >
-              <template #name="{ row }">
+              <template #name="{ row: item }">
```

Comment:
**Suggestion**: Could we add the explicit type annotations for the rows here so we have a better sense of safety that what we’re accessing here is correct?

```suggestion
              <template #name="{ row: item }: { row: ZoneIngressOverview }">
```

Reply:
Oh that's weird I thought we'd addressed this centrally?

When we made AppCollection generic was there a reason why we didn't pass that through:

https://github.com/kumahq/kuma-gui/blob/f50cd686172fa54593dc5a03bb1da772be012d51/src/app/application/components/app-collection/AppCollection.vue#L88

i.e. make the above

```
:row="row as Row"
```

Reply:
Yeah, we can do that. Then that would be working centrally.

Reply:
Lemme make a separate PR, then we can rebase that to all these data layer PRs

Reply:
x ref https://github.com/kumahq/kuma-gui/pull/1889